src/care/algorithm_impl.h

```diff
@@ -1515,21 +1515,79 @@ CARE_INLINE void ArrayCopy(RAJA::seq_exec,
    } CARE_SEQUENTIAL_LOOP_END
 }
 
+/************************************************************************
+ * Function  : ArrayCopy
```

Comment:
Documentation: change "ArrayCopy" to "ArrayDup" to match the function name.

Reply:
Fixed.